decidim-core/app/controllers/decidim/devise/sessions_controller.rb

```diff
@@ -6,6 +6,14 @@ class SessionsController < ::Devise::SessionsController
       include Decidim::NeedsOrganization
       include Decidim::LocaleSwitcher
       layout "application"
+
+      def after_sign_in_path_for(user)
+        if user.is_a?(User) && user.sign_in_count == 1 && Decidim.authorization_handlers.any?
```

Comment:
Are you sure you need to check the user's actually an `User`? AFAIK system's using a different `sessions_controller`.